data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,783 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+ <teiHeader xml:lang="eng">
+  <fileDesc>
+   <titleStmt>
+    <title>Aristarchus of Samos, the Ancient Copernicus: a History of Greek Astronomy to
```

Comment:
The `<title>` element in the `<titleStmt>` should be the title of the work, not the title of the volume. According to the TLG, the title of this work is "De magnitudinibus et distantiis solis et lunae". Make sure also to put an xml:lang="lat" attribute on the `<title>` element.